app/config/question-bank.js

```diff
@@ -228,22 +228,17 @@ const questionBank = {
           title: 'What is the legal status of the business?',
           pageTitle: '',
           backUrlObject: {
-            dependentQuestionYarKey: 'applicant',
-            dependentAnswerKeysArray: ['applicant-A1'],
+            dependentQuestionYarKey: ['projectSubject', 'applicant'],
+            dependentAnswerKeysArray: ['project-subject-A2', 'applicant-A1'],
             urlOptions: {
-              thenUrl: 'applicant',
-              elseUrl: 'project-subject'
+              thenUrl: ['project-subject', 'applicant'],
+              elseUrl: 'country'
             }
           },
-          nextUrl: 'country',
-          url: 'legal-status',
           baseUrl: 'legal-status',
-          preValidationObject: {
```

Comment:
this is the correct object btw. Please remove **preValidationKeys** and re add this object in